.github/workflows/erase_db.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: |
           docker volume rm dspace-${{env.INSTANCE}}_pgdata dspace-${{env.INSTANCE}}_assetstore
       - name: redeploy
-        if: 'false'
+        if: '!cancelled()'
```

Comment:
as above, should be false, not !cancelled()

Reply:
fixed